src/ui.cpp

```diff
@@ -534,23 +538,31 @@ UpdateDialog::UpdateDialog()
     m_buttonSizer = new wxBoxSizer(wxHORIZONTAL);
 
     m_updateButtonsSizer = new wxBoxSizer(wxHORIZONTAL);
-    m_updateButtonsSizer->Add
-                          (
+
+
+    m_SkipVersionButtonSizer = new wxBoxSizer(wxHORIZONTAL);
+
+    m_SkipVersionButtonSizer->Add(
                             new wxButton(this, ID_SKIP_VERSION, _("Skip this version")),
-                            wxSizerFlags().Border(wxRIGHT, PX(20))
-                          );
-    m_updateButtonsSizer->AddStretchSpacer(1);
-    m_updateButtonsSizer->Add
-                          (
+                            wxSizerFlags()
+    );
+
+    m_RemindLaterButtonSizer = new wxBoxSizer(wxHORIZONTAL);
+    m_RemindLaterButtonSizer->Add(
```

Comment:
You don't need one sizer per control, you can use the `SHOW()` and `HIDE()` macros for individual controls too, not just sizers.